tao_compiler/mlir/disc/transforms/tests/scatter-legalize.mlir

```diff
@@ -0,0 +1,24 @@
+// RUN: disc-opt -split-input-file -disc-hlo-legalize-to-lhlo -hlo-legalize-to-lhlo \
+// RUN:  -canonicalize -split-input-file %s -o - | FileCheck %s
+
+// -----
+
+
+// CHECK-LABEL: @test_scatterop_lowering
+// CHECK-SAME: %[[ARG0:.*]]: memref<32000x4096xf32>, %[[ARG1:.*]]: memref<8193x1xi64>, %[[ARG2:.*]]: memref<8192x4096xf32>
+func.func @test_scatterop_lowering(%arg0: tensor<32000x4096xf32>, %arg1: tensor<8192x1xi64>, %arg2: tensor<8192x4096xf32>) -> tensor<32000x4096xf32>
+  attributes {
+    tf.entry_function = {
+      input_placements = "cpu, cpu, cpu",
+      inputs = "input0, input1, input2",
+      output_placements = "cpu",
+      outputs = "output0"
+    }} {
+    %2 = "mhlo.scatter"(%arg0, %arg1, %arg2) ({
```

Comment:
this ut is too simple, please check the IR of scatter op lowing.

Reply:
Fixed.